project-folder/CNN.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<p>you don't actually need this step since the model is already saved in memory</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>